src/repr/src/row.rs

```diff
@@ -895,6 +895,13 @@ impl Row {
         row.packer().extend(slice.iter());
         row
     }
+
+    /// Returns the total amount of bytes used by this row.
+    pub fn byte_len(&self) -> usize {
+        let heap_size = self.data.capacity().saturating_sub(self.data.inline_size());
```

Comment:
IMO this should call `length()`, not `capacity()`, because there's not necessarily any reason to believe that we will have the same amount of unused capacity when re-creating it on the other side as we do here.

Reply:
Updated.